src/routes/safe/components/Settings/ManageOwners/RemoveOwnerModal/screens/Review/index.tsx

```diff
@@ -248,18 +252,19 @@ export const ReviewRemoveOwnerModal = ({
             <Button minHeight={42} minWidth={140} onClick={onClickBack}>
```

Comment:
we have to remove this.

![add-owner-back-button](https://user-images.githubusercontent.com/622217/117351847-97c33180-ae84-11eb-98fd-c9390fc65748.png)

Reply:
oops! fixed, thanks!